### PR TITLE
added cnn results

### PR DESCRIPTION
**Pull request by craftamap**

### CODE REVIEW

**Review by AnnaGsell — APPROVED** [review]

**Review by diehmlukas — APPROVED** [review]

File: chapters/results_and_discussion.tex [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+Regarding the soHappy smile detection machine learning model, the following
+result is present: After the training was finished, it was evaluated with the
+test data. As Figure \ref{fig:training_result} shows, the accuracy detecting a
+smile as a smile is about 98\%, with an overall evaluation accuracy of about 
+94.5\%.
+
+\begin{figure}
+  \includegraphics[width=\linewidth]{figures/training_result.png}
+  \caption{Headmap }
```
Comment (limkokyew) [inline chapters/results_and_discussion.tex]: Maybe a slightly more detailed description would be preferable. A period is also missing.

File: chapters/results_and_discussion.tex [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+  \label{fig:training_result}
+\end{figure}
+
+While those numbers look promising, using this model in production shows
+room for improvement. Sometimes it is not enough to smile with a closed
+mouth, instead an open mouth is required.
```
Comment (limkokyew) [inline chapters/results_and_discussion.tex]: Didn't we write "[...] closed mouth. Instead, an open mouth [...]"?

File: chapters/results_and_discussion.tex [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+
+In the future, the soHappy model could be improved by improving the machine 
+learning architecture. Also an bigger dataset including more photographs of
```
Comment (limkokyew) [inline chapters/results_and_discussion.tex]: Comma after "Also".
"an bigger" -> "a bigger"

File: chapters/results_and_discussion.tex [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+people smiling with mouth closed could significantly improve the performance.
```
Comment (limkokyew) [inline chapters/results_and_discussion.tex]: Omit "the"

**Review by mac641 — APPROVED** [review]

**Pull request merged**